arch/x86/Makefile

```diff
@@ -138,6 +138,7 @@ else
 
         KBUILD_CFLAGS += -mno-red-zone
         KBUILD_CFLAGS += -mcmodel=kernel
+        KBUILD_RUSTFLAGS += -Cno-redzone=y
```

Comment:
Does this also apply to other targets? Or is x86 the only for which the abi has redzones?

Reply:
No, this will get only applied to x86. There are other architectures with redzones, e.g. within the kernel powerpc64.